scss/partials/buttons.scss

```diff
@@ -0,0 +1,8 @@
+@mixin brand-button($padding) {
+  text-decoration: none;
+  padding: $padding 2*$padding;
+  background: $brand-blue;
+  color: #FFF;
+  margin-left: $padding * -2;
+  @include border-radius(2*$padding);
```

Comment:
I'll remove this whole thing.  This styling was for the rounded "Download" button in the original design you supplied.  However, since the button has been removed we no longer need the "brand-button" mixin.  I'll remove it.